lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -302,7 +316,9 @@ export async function getPrList(refreshCache?: boolean): Promise<Pr[]> {
     };
     if (!config.ignorePrAuthor) {
       searchParams['role.1'] = 'AUTHOR';
-      searchParams['username.1'] = config.username;
+      if(config.username){
```

Comment:
Can the username be "learned" using the API? It's possible in other platforms to do a "whoami" type query to the API

Reply:
no, afaik the point is to be anonymous. the tokens are for a repository and not for a user. They are no PAT.

Reply:
please check what's the author for PRs created by such a token.